### PR TITLE
Pick up astrolabe changes about updating vddk libs to 7.0.2 from 7.0

### PR DESCRIPTION
**Pull request by xinyanw409**
**What this PR does / why we need it**:
This PR pickes up the astrolabe change about upgrading VDDK libs from 7.0 to 7.0.1
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
This PR includes the following changes:
1. Changed Action YAML to make sure images built from github action CI contains 7.2 VDDK libs
2. Update go.mod to pick up latest astrolabe change
3.  Add size to NewProtectedEntityInfo input to keep consistency with astrolabe change

Test on vanilla: https://container-dp.svc.eng.vmware.com/job/Container_Precheck_Velero/1079/
Guest: https://container-dp.svc.eng.vmware.com/view/CNS-DP/job/Velero-Pipeline-WCP/566/
Supervisor: https://container-dp.svc.eng.vmware.com/view/CNS-DP/job/Velero-Pipeline-WCP/568/

```release-note
Upgraded VDDK library in dependency from vSphere 7.0 to 7.0U2
```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/542/

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/543/

File: pkg/paravirt/paravirt_protected_entity.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func (this ParaVirtProtectedEntity) GetInfo(ctx context.Context) (astrolabe.Prot
 		}
 	}
 
-	return astrolabe.NewProtectedEntityInfo(this.id, name, this.data, this.metadata, this.combined, []astrolabe.ProtectedEntityID{}), nil
+	return astrolabe.NewProtectedEntityInfo(this.id, name, -1, this.data, this.metadata, this.combined, []astrolabe.ProtectedEntityID{}), nil
```
Comment (lintongj) [inline pkg/paravirt/paravirt_protected_entity.go:49]: Please think through what does -1 means here and why it is safe to use -1 here. Then add some comments above this line.

Comment (xinyanw409) [reply pkg/paravirt/paravirt_protected_entity.go:50]: Add one line comments.

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/544/

**Comment by xinyanw409** [comment]
> Changes looks good but there are still two other comments.
> 
> 1. Please update the release notes in the PR description accordingly. It might be a breaking change. So, we should not leave it as NONE. For example,
> 
> ```
> Upgraded VDDK libraries in the dependency of plugin from vSphere 7.0 to vSphere 7.0U2.
> ```
> 
> 1. Please make sure regression test on different cluster flavors get passed. And explicitly call it out in your PR description.

Added note and test link.

**Comment by lintongj** [comment]
> > Changes looks good but there are still two other comments.
> > 
> > 1. Please update the release notes in the PR description accordingly. It might be a breaking change. So, we should not leave it as NONE. For example,
> > 
> > ```
> > Upgraded VDDK libraries in the dependency of plugin from vSphere 7.0 to vSphere 7.0U2.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > 1. Please make sure regression test on different cluster flavors get passed. And explicitly call it out in your PR description.
> 
> Added note and test link.

The images built from CNSDP-CI pipeline are still using VDDK 7.0 libs if you didn't explicitly request QE folks to make a change in pipeline script. Meanwhile, QE folks would not like to make the change until we actually test through all regression tests. 

So, please make images that actually pick up VDDK 7.0U2 libs and run through all regression tests.

**Comment by xinyanw409** [comment]
> > > Changes looks good but there are still two other comments.
> > > 
> > > 1. Please update the release notes in the PR description accordingly. It might be a breaking change. So, we should not leave it as NONE. For example,
> > > 
> > > ```
> > > Upgraded VDDK libraries in the dependency of plugin from vSphere 7.0 to vSphere 7.0U2.
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > 
> > > 1. Please make sure regression test on different cluster flavors get passed. And explicitly call it out in your PR description.
> > 
> > 
> > Added note and test link.
> 
> The images built from CNSDP-CI pipeline are still using VDDK 7.0 libs if you didn't explicitly request QE folks to make a change in pipeline script. Meanwhile, QE folks would not like to make the change until we actually test through all regression tests.
> 
> So, please make images that actually pick up VDDK 7.0U2 libs and run through all regression tests.

Updated regression tests link.

**Review by lintongj — APPROVED** [review]

**Pull request merged**